src/components/labels/Label.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Icon, {COLOR as ICON_COLOR, TYPE as ICON_TYPE} from '../icons/Icon';
+
+const SIZE = {
+  NORMAL: {CLASS_NAME: 'normal', ICON_SIZE: 16},
+  SMALL: {CLASS_NAME: 'small', ICON_SIZE: 14},
+  LARGE: {CLASS_NAME: 'large', ICON_SIZE: 24}
+};
+
+const Label = ({text, number, iconType, iconColor,
+  size = SIZE.NORMAL, secondary, unstyled, emphasised, elementsToTop
+}) => {
+
+  const labelClass = classNames('sg-label', {
+    [`sg-label--${size.CLASS_NAME}`]: size !== SIZE.NORMAL,
+    'sg-label--secondary': secondary,
+    'sg-label--unstyled': unstyled,
+    'sg-label--emphasised': emphasised,
+    'sg-label--elements-to-the-top': elementsToTop
+  });
+
+  if (number) {
+    return <div className={labelClass}>
+      <div className="sg-label__icon">
+        <Icon type={iconType} color={iconColor} size={size.ICON_SIZE}/>
+      </div>
+      <div className="sg-label__text">{text}</div>
+      <div className="sg-label__number">{number}</div>
+    </div>;
+  }
+
+  return <div className={labelClass}>
+    <div className="sg-label__icon">
+      <Icon type={iconType} color={iconColor} size={size.ICON_SIZE}/>
+    </div>
+    <div className="sg-label__text">{text}</div>
+  </div>;
```

Comment:
this code is almost the same as above - maybe we can do sth like {name ? <div>bla</div> : null}. @matikrk @aju WDYT?